ceph/ceph_admin/registry_login.py

```diff
@@ -0,0 +1,61 @@
+"""Cephadm registry login CLI Command."""
```

Comment:
Likewise, this can be a separate commit as it is introducing a new functionality.

Reply:
Done.